sdk/go/kms_test.go

```diff
@@ -0,0 +1,116 @@
+package ehsm
+
+import (
+	"encoding/base64"
+	"testing"
+)
+
+const EH_DIGEST = "31b05f630675290977311f476bafba9406539c73aaa773f27fb"
+
+func TestECSignVerify(t *testing.T) {
+	client, err := NewClient()
+	if err != nil {
+		t.Error(err)
+	}
+	keyspec := []string{"EH_EC_P224", "EH_EC_P256", "EH_EC_P256K", "EH_EC_P384", "EH_EC_P521"}
+	for _, keyspec := range keyspec {
+		keyid, err := client.CreateKey(keyspec, "EH_INTERNAL_KEY", "EH_KEYUSAGE_SIGN_VERIFY")
+		if err != nil {
+			t.Error(err)
+		}
+		signature, err := client.Sign(keyid, base64.StdEncoding.EncodeToString([]byte(EH_DIGEST)), "EH_RAW", "EH_PAD_NONE", "EH_SHA_256")
+		if err != nil {
```

Comment:
you're misunderstanding the usage of "EH_RAW" and "EH_DIGEST"

Reply:
still not expected.

EH_RAW means you need to input a message with raw data, such as msg = "test for signature" in base64 encoding.

EH_DIGEST means you need to hash the msg with the digest function(SHA224, SHA256, SHA384, SHA512) to get its digest firstly, such as msg_digest = SHA256(msg), then base64 encode.

so they are two different test cases.